types/openpgp/index.d.ts

```diff
@@ -39,6 +40,8 @@ export interface EncryptOptions {
 export interface EncryptedMessage {
     data?: string,
     message?: message.Message,
+    signature?: string | ReadableStream | Signature // TODO add NodeStream
```

Comment:
Did you mean to do those todos now or later?

Reply:
We could do leave them to be done in a future PR, if you agree!